# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../boxbackup/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- boxbackup-0.10/bin/bbstoreaccounts/bbstoreaccounts.cpp.vanilla	2008-04-13 18:23:00.000000000 +0200
+++ boxbackup-0.10/bin/bbstoreaccounts/bbstoreaccounts.cpp	2008-04-13 18:23:08.000000000 +0200
@@ -51,6 +51,7 @@
 #include <stdio.h>
 #include <sys/types.h>
 #include <limits.h>
+#include <string.h>
 #include <vector>
 #include <algorithm>
 
--- boxbackup-0.10/bin/bbackupctl/bbackupctl.cpp.vanilla	2008-04-13 18:24:38.000000000 +0200
+++ boxbackup-0.10/bin/bbackupctl/bbackupctl.cpp	2008-04-13 18:24:46.000000000 +0200
@@ -48,6 +48,7 @@
 #include "Box.h"
 
 #include <stdio.h>
+#include <stdlib.h>
 
 #ifdef HAVE_UNISTD_H
 	#include <unistd.h>
--- boxbackup-0.10/bin/bbackupquery/bbackupquery.cpp.vanilla	2008-04-13 18:23:26.000000000 +0200
+++ boxbackup-0.10/bin/bbackupquery/bbackupquery.cpp	2008-04-13 18:23:37.000000000 +0200
@@ -50,6 +50,7 @@
 #ifdef HAVE_UNISTD_H
 	#include <unistd.h>
 #endif
+#include <stdlib.h>
 #include <stdio.h>
 #include <sys/types.h>
 #ifdef HAVE_LIBREADLINE
--- boxbackup-0.10/lib/backupclient/BackupClientFileAttributes.cpp.vanilla	2008-04-13 18:19:38.000000000 +0200
+++ boxbackup-0.10/lib/backupclient/BackupClientFileAttributes.cpp	2008-04-13 18:20:30.000000000 +0200
@@ -53,8 +53,8 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
-#include <string.h>
 #include <limits.h>
+#include <cstring>
 #include <algorithm>
 #include <new>
 #include <vector>
--- boxbackup-0.10/lib/backupclient/BackupStoreFileDiff.cpp.vanilla	2008-04-13 18:21:32.000000000 +0200
+++ boxbackup-0.10/lib/backupclient/BackupStoreFileDiff.cpp	2008-04-13 18:21:50.000000000 +0200
@@ -50,6 +50,8 @@
 #include <new>
 #include <map>
 
+#include <string.h>
+
 #ifdef HAVE_TIME_H
 	#include <time.h>
 #elif HAVE_SYS_TIME_H
--- boxbackup-0.10/lib/backupclient/BackupStoreFile.h.vanilla	2008-04-13 18:17:13.000000000 +0200
+++ boxbackup-0.10/lib/backupclient/BackupStoreFile.h	2008-04-13 18:17:35.000000000 +0200
@@ -52,6 +52,7 @@
 #include "BackupClientFileAttributes.h"
 #include "BackupStoreFilename.h"
 
+#include <stdlib.h>
 #include <memory>
 
 typedef struct 
--- boxbackup-0.10/lib/backupclient/BackupStoreFileEncodeStream.cpp.vanilla	2008-04-13 18:20:55.000000000 +0200
+++ boxbackup-0.10/lib/backupclient/BackupStoreFileEncodeStream.cpp	2008-04-13 18:21:16.000000000 +0200
@@ -62,6 +62,8 @@
 
 #include "MemLeakFindOn.h"
 
+#include <string.h>
+
 using namespace BackupStoreFileCryptVar;
 
 
--- boxbackup-0.10/lib/raidfile/RaidFileRead.cpp.vanilla	2008-04-13 18:22:21.000000000 +0200
+++ boxbackup-0.10/lib/raidfile/RaidFileRead.cpp	2008-04-13 18:22:37.000000000 +0200
@@ -49,6 +49,7 @@
 
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdlib.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <sys/stat.h>
--- boxbackup-0.10/lib/common/WaitForEvent.h.vanilla	2008-04-13 18:18:10.000000000 +0200
+++ boxbackup-0.10/lib/common/WaitForEvent.h	2008-04-13 18:18:27.000000000 +0200
@@ -58,6 +58,8 @@
 	#endif
 #endif
 
+#include <stdlib.h>
+
 #include "CommonException.h"
 
 #include "MemLeakFindOn.h"
--- boxbackup-0.10/lib/common/Configuration.cpp.vanilla	2008-04-13 18:18:48.000000000 +0200
+++ boxbackup-0.10/lib/common/Configuration.cpp	2008-04-13 18:19:02.000000000 +0200
@@ -49,6 +49,7 @@
 
 #include <stdlib.h>
 #include <limits.h>
+#include <string.h>
 
 #include "Configuration.h"
 #include "CommonException.h"
